routes/EditUser.js

```diff
@@ -2,13 +2,15 @@ var express = require("express");
 var router = express.Router();
 var Q = require("q");
 
-router.route('/:username/:firstName/:lastName/:perms/:isConfirmed').get(function(req,res) {
+router.route('/:username/:firstName/:lastName/:perms/:requestDelete').get(function(req,res) {
 
     var username = req.params.username;
     var firstName = req.params.firstName;
     var lastName = req.params.lastName;
     var perms = req.params.perms;
-    var isConfirmed = req.params.isConfirmed;
+    var requestDelete = req.params.requestDelete;
```

Comment:
You are removing isConfirmed, but you didnt remove the use of it in args. Did you actually compile and run this? It should have thrown an error about isConfirmed not being defined.